__init__.py

```diff
@@ -42,51 +41,265 @@ class SetupState(str, Enum):
     PAIRING = "pairing"
 
 
+class PairingMode(str, Enum):
+    VOICE = "voice"  # voice only
+    GUI = "gui"  # gui - click buttons
+    HYBRID = "hybrid"  # gui but no input capabilities - select with voice
+
+
+class BackendType(str, Enum):
+    OFFLINE = "offline"
+    PERSONAL = "personal"
+    SELENE = "selene"
+
+
+class SetupManager:
+    """ helper class to perform setup actions"""
+
+    def __init__(self, bus):
+        self.bus = bus
+        self.config_lock = Lock()
+        self._offline_male = {
+            "module": "ovos-tts-plugin-mimic",
+            "ovos-tts-plugin-mimic": {"voice": "ap"}
```

Comment:
@AIIX can you refactor the GUI to populate buttons from these options? both for TTS and STT